src/strings/PowerFxResources.de-DE.resx

```diff
@@ -3297,7 +3297,7 @@
     
   </data>
   <data name="ErrRemoveAllArg" xml:space="preserve">
-    <value>Falls angegeben, muss das letzte Argument „Alle sein. Gibt es einen Tippfehler?</value>
+    <value>Falls angegeben, muss das letzte Argument „Alle“ sein. Gibt es einen Tippfehler?</value>
```

Comment:
oof. the last argument should be `All`, Locked. I'll put up a PR fixing this in en-US